core/user/user.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package user
+
+import (
+	"fmt"
+	"time"
+)
+
+type User struct {
+	CreatedAt   time.Time
+	DisplayName string
+	Name        string
+}
+
+// UserPassword type attempts to protect it against printouts
```

Comment:
@cderici,

I think we need to change the wording here a little.

I don't think we want to talk about "over the wire" as that is a problem of another layer.


Maybe we should just say "UserPassword hides a plain text user password from being directly read outside of this package and also makes a best effort attempt to stop common cases of this type from being printed to any log or stdout sources."

Reply:
ignoring based on convo w/ @tlm